master/buildbot/test/unit/test_worker_docker.py

```diff
@@ -29,6 +29,12 @@
 from buildbot.worker import docker as dockerworker
 
 
+_thd_parse_volumes__orig = dockerworker.DockerLatentWorker._thd_parse_volumes
+def _thd_parse_volumes(self, volumes):
+    self._test_volumes, self._test_binds = retval = _thd_parse_volumes__orig(self, volumes)
```

Comment:
This patching looks a bit ugly to me. I think it would be better to assert that create_container has been called properly.

You can modify the fake docker in order to record any call to create container, and test this instead. (this probably should have been tested like this from the start)